packages/world/gas-report.json

```diff
@@ -231,67 +231,73 @@
     "file": "test/StandardDelegationsModule.t.sol",
     "test": "testCallFromCallboundDelegation",
     "name": "register a callbound delegation",
-    "gasUsed": 122345
+    "gasUsed": 128059
   },
   {
     "file": "test/StandardDelegationsModule.t.sol",
     "test": "testCallFromCallboundDelegation",
     "name": "call a system via a callbound delegation",
-    "gasUsed": 44047
+    "gasUsed": 46208
   },
   {
     "file": "test/StandardDelegationsModule.t.sol",
     "test": "testCallFromTimeboundDelegation",
     "name": "register a timebound delegation",
-    "gasUsed": 116594
+    "gasUsed": 121016
   },
   {
     "file": "test/StandardDelegationsModule.t.sol",
     "test": "testCallFromTimeboundDelegation",
     "name": "call a system via a timebound delegation",
-    "gasUsed": 34791
+    "gasUsed": 36952
+  },
+  {
+    "file": "test/SystemCall.t.sol",
+    "test": "testGasSystemCallEvent",
+    "name": "emit a system call event",
+    "gasUsed": 3158
```

Comment:
Unfortunately it seems like emitting this event is pretty significant. With the current design we'd have to emit it twice per system call.

Reply:
why is it so expensive? it seems like the data is not that big